priv/repo/seeds/orders.exs

```diff
@@ -0,0 +1,128 @@
+defmodule Store.Repo.Seeds.Orders do
+  alias Store.Repo
+  alias Store.Inventory.Order
+
+  def run do
+    seed_orders()
+  end
+
+  def seed_orders do
```

Comment:
I think this can be accomplished with a loop and an `Ecto.UUID.generate()` for the id's. What do you think? 🙏🏼